accepted/2021/net6.0-tfms/net6.0-tfms.md

```diff
@@ -0,0 +1,110 @@
+# .NET 6.0 Target Frameworks
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+In [.NET 5.0][net5.0] we defined a new syntax for target frameworks (TFM) that
+allows us to avoids tools and humans to use a decoder ring, such as the .NET
+Standard version table, to figure out what is compatible with what. In this
+document, we're listing which TFMs we're adding in .NET 6.0.
+
+## Scenarios and User Experience
+
+TBD
+
+## Requirements
+
+### Goals
+
+* Support the definition of frameworks relevant for Xamarin development
+
+### Non-Goals
+
+* Re-design the TFM syntax
+
+## Stakeholders and Reviewers
+
+* MSBuild / SDK team
+* NuGet team
+* Project system team
+* Xamarin team
+
+## Design
+
+| TFM                | Compatible With                          |
+|--------------------|------------------------------------------|
+| net6.0             | (subsequent version of net5.0)           |
+| net6.0-windows     | (subsequent version of net5.0-windows)   |
+| net6.0-android     | xamarin.android                          |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-ios         | xamarin.ios                              |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-macos       | xamarin.mac                              |
+|                    | (+everything else inherited from net6.0) |
+| net6.0-maccatalyst | xamarin.ios                              |
```

Comment:
A related question is whether it's possible to add compatibility for other TFMs in the future or not: can `net7.0-ios` be made compatible with `net7.0-maccatalyst` if we want to at that point, even if `net6.0-ios` is not compatible with `net6.0-maccatalyst`?

Reply:
You can add new compat mappings in new versions of the TFM. You can only add compa mappings to existing TFM versions if those take effect when nothing is compatible (otherwise it would be a breaking change).